### PR TITLE
avoid potential bug for auto-gptq 0.8

### PR DESCRIPTION
**Pull request by wenhuach21**

### CODE REVIEW

File: auto_round/export/export_to_autoround/export.py [hunk 1]
```diff
@@ -130,7 +81,7 @@ def dynamic_import_quantLinear_for_packing(backend, bits, group_size, sym):
     elif "gptq" in backend:
         return get_autogptq_packing_qlinear(backend, bits, group_size, sym)
     else:
-        assert False, f"only support gptq and autoround backend"
+        assert False, f"only support auto_gptq, auto_awq,and auto_round backend"
```
Comment (WeiweiZhang1) [inline auto_round/export/export_to_autoround/export.py]: watch the blank space in log

File: auto_round/utils.py [hunk 2]
```diff
@@ -806,3 +826,83 @@ def dynamic_import_inference_linear(backend, bits, group_size, sym):
...
+    else:
+        QuantLinear = dynamically_import_QuantLinear(
+            use_triton=use_triton,
+            desc_act=False,
+            group_size=group_size,
+            bits=bits,
+            disable_exllama=disable_exllamav1,
+            disable_exllamav2=disable_exllamav2,
+            use_qigen=use_qigen,
+            use_marlin=not disable_marlin,
```
Comment (WeiweiZhang1) [inline auto_round/utils.py:906]: use_tritonv2=use_tritonv2 is not needed here?

**Review by WeiweiZhang1 — APPROVED** [review]

File: auto_round/utils.py (hunk 2 above)
Comment (wenhuach21) [reply auto_round/utils.py:906]: not supported in 0.7.1

Comment (WeiweiZhang1) [reply auto_round/utils.py:906]: this is 0.7.1 above version

Comment (wenhuach21) [reply auto_round/utils.py:906]: no need here, this is for packing, v2 and v1 are using the same packing code

**Pull request merged**